db/auto_roll_logger_test.cc

```diff
@@ -111,45 +118,43 @@ void AutoRollLoggerTest::RollLogFileBySizeTest(AutoRollLogger* logger,
   ASSERT_TRUE(message_size == logger->GetLogFileSize());
 }
 
-uint64_t AutoRollLoggerTest::RollLogFileByTimeTest(
-    AutoRollLogger* logger, size_t time, const std::string& log_message) {
-  uint64_t expected_create_time;
-  uint64_t actual_create_time;
+void AutoRollLoggerTest::RollLogFileByTimeTest(Env* _env,
```

Comment:
can we give it a name that distinguishes it from `env`? or maybe `s/env/DefaultEnv()/` since that static variable's not really necessary.